theano/sandbox/gpuarray/dnn.py

```diff
@@ -40,9 +40,9 @@ def raise_no_cudnn(msg="CuDNN is required for convolution and pooling"):
 
 def _dnn_check_compile():
     preambule = """
-#include <stdio.h>
-#include <cudnn.h>
-#include <cudnn_helper.h>
+# include <stdio.h>
+# include <cudnn.h>
+# include <cudnn_helper.h>
```

Comment:
why add a space here? We don't do that elsewhere, so revert that for consistency.